---
File: README.md
Lecture: Advanced Bioinformatics 1
Professor: Martin Steinegger (martin.steinegger@snu.ac.kr)
TA: Sooyoung Cha, Victor Mihaila (steineggerlab.ta.2025@gmail.com)
Last Modified: 2025-07-25
---

# Advanced Bioinformatics 1
# The paper pool for Advanced Bioinformatics 2025

## Annotation
1. Prokka [paper](https://academic.oup.com/bioinformatics/article/30/14/2068/2390517), [github](https://github.com/tseemann/prokka)
2. Augustus [paper](https://academic.oup.com/bioinformatics/article/27/6/757/234821), [github](https://github.com/Gaius-Augustus/Augustus)
3. Maker [paper](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-491), [homepage](https://www.yandell-lab.org/software/maker.html)
4. Prodigal [paper](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-11-119), [github](https://github.com/hyattpd/Prodigal)
5. Bakta [paper](https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000685), [github](https://github.com/oschwengers/bakta)
6. DFAST [paper](https://academic.oup.com/bioinformatics/article/34/6/1037/4587587), [github](https://github.com/nigyta/dfast_core)
7. EggNOG [paper1](https://academic.oup.com/nar/article/47/D1/D309/5173662), [paper2](https://academic.oup.com/mbe/article/38/12/5825/6379734) [github](https://github.com/eggnogdb/eggnog-mapper), [homepage](http://eggnog-mapper.embl.de/)
8. Alphagenome [paper](https://www.biorxiv.org/content/10.1101/2025.06.25.661532v1.full), [github](https://github.com/google-deepmind/alphagenome)

## Genome comparison
1. mummer1-4 [paper](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005944), [github](https://github.com/mummer4/mummer)
2. mash [paper](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0997-x), [github](https://github.com/marbl/mash)
3. skani [paper](https://www.nature.com/articles/s41592-023-02018-3), [github](https://github.com/bluenote-1577/skani)

## Multiple sequence alignment (MSA)
1. clustalomega [paper](https://www.embopress.org/doi/full/10.1038/msb.2011.75), [github](https://github.com/GSLBiotech/clustal-omega), [homepage](http://www.clustal.org/omega/)
2. mafft [paper](https://academic.oup.com/mbe/article/30/4/772/1073398), [github](https://github.com/GSLBiotech/mafft), [homepage](https://mafft.cbrc.jp/alignment/software/)
3. muscle [paper1](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-5-113), [paper2](https://academic.oup.com/nar/article/32/5/1792/2380623), [github](https://github.com/rcedgar/muscle), [homepage](https://www.drive5.com/muscle/)
4. T-Coffee [paper1](https://www.sciencedirect.com/science/article/pii/S0022283600940427), [paper2](https://academic.oup.com/nar/article/39/suppl_2/W13/2505784), [github](https://github.com/cbcrg/tcoffee), [homepage](https://tcoffee.crg.eu/)
5. Kalign 3 [paper](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-6-298), [paper2](https://pmc.ncbi.nlm.nih.gov/articles/PMC2647288/), [paper3](https://academic.oup.com/bioinformatics/article/36/6/1928/5607735), [github](https://github.com/TimoLassmann/kalign)

## Homology search
1. MMseqs2 [paper](https://www.nature.com/articles/nbt.3988), [github](https://github.com/soedinglab/MMseqs2)
2. Diamond1+2 [paper1](https://www.nature.com/articles/s41592-021-01101-x), [paper2](https://www.nature.com/articles/nmeth.3176), [github](https://github.com/bbuchfink/diamond)
3. HHblits [paper](https://www.nature.com/articles/nmeth.1818), [github](https://github.com/soedinglab/hh-suite)

## Transcript
1. stringtie1+2 [paper1](https://www.nature.com/articles/nbt.3122), [paper2](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1910-1), [github](https://github.com/gpertea/stringtie), [homepage](http://ccb.jhu.edu/software/stringtie/)
2. kallisto [paper](https://www.nature.com/articles/nbt.3519), [github](https://github.com/pachterlab/kallisto), [homepage](https://pachterlab.github.io/kallisto/)
3. salmon [paper](https://www.nature.com/articles/nmeth.4197), [github](https://github.com/COMBINE-lab/salmon), [homepage](https://combine-lab.github.io/salmon/)
4. RSEM [paper](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-323), [github](https://github.com/deweylab/RSEM)

## Metagenomics
1. metaphlan3 [metaphlan1_paper](https://www.nature.com/articles/nmeth.2066), [metaphlan2_paper](https://www.nature.com/articles/nmeth.3589), [metaphlan3_paper](https://elifesciences.org/articles/65088), [github](https://github.com/biobakery/MetaPhlAn), [homepage](https://huttenhower.sph.harvard.edu/metaphlan3/)
2. kraken2 [paper](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1891-0), [github](https://github.com/DerrickWood/kraken2)
3. Kaiju [paper](https://www.nature.com/articles/ncomms11257), [github](https://github.com/bioinformatics-centre/kaiju)
4. MetaMaps [paper](https://www.nature.com/articles/s41467-019-10934-2), [github](https://github.com/DiltheyLab/MetaMaps)
5. Metabuli [paper](https://www.nature.com/articles/s41592-024-02273-y), [github](https://github.com/steineggerlab/Metabuli)

## Phylogenetics
1. raxml [paper](https://academic.oup.com/bioinformatics/article/30/9/1312/238053), [github](https://github.com/stamatak/standard-RAxML), [homepage](https://cme.h-its.org/exelixis//web/software/raxml/index.html)
2. fasttree1,2 [paper1](https://academic.oup.com/mbe/article/26/7/1641/1128976),[paper2](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0009490), [homepage](http://www.microbesonline.org/fasttree/)
3. iqtree1,2,3 [iqtree1_paper](https://academic.oup.com/mbe/article/37/5/1530/5721363), [iqtree2_paper](https://academic.oup.com/mbe/article/37/5/1530/5721363), [iqtree3_paper](https://ecoevorxiv.org/repository/view/8916/), [github](https://github.com/iqtree/iqtree3), [homepage](http://www.iqtree.org/)
4. Unicore [paper](https://academic.oup.com/gbe/article/17/6/evaf109/8155201), [github](https://github.com/steineggerlab/unicore)
5. orthofinder [paper](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1832-y), [github](https://github.com/davidemms/OrthoFinder)

## RNA structure prediction
1. viennarna [paper](https://almob.biomedcentral.com/articles/10.1186/1748-7188-6-26), [github](https://github.com/ViennaRNA/ViennaRNA), [homepage](https://www.tbi.univie.ac.at/RNA/)
2. LinearFold [paper](https://academic.oup.com/bioinformatics/article/35/14/i295/5529205?login=true), [github](https://github.com/LinearFold/LinearFold)
3. UFold [paper](https://academic.oup.com/nar/article/50/3/e14/6430845?login=true), [github](https://github.com/uci-cbcl/UFold)
4. MXfold2 [paper](https://www.nature.com/articles/s41467-021-21194-4), [github](https://github.com/mxfold/mxfold2)
5. RoseTTAFoldNA [paper](https://www.nature.com/articles/s41592-023-02086-5), [github](https://github.com/uw-ipd/RoseTTAFold2NA)
6. RhoFold [paper](https://www.nature.com/articles/s41592-024-02487-0), [github](https://github.com/ml4bio/RhoFold)

## Protein structure
1. AlphaFold2 [paper](https://www.nature.com/articles/s41586-021-03819-2), [github](https://github.com/deepmind/alphafold)
2. RoseTTAFold [paper](https://www.science.org/doi/abs/10.1126/science.abj8754), [github](https://github.com/RosettaCommons/RoseTTAFold)
3. Colabfold [paper](https://www.nature.com/articles/s41592-022-01488-1), [github](https://github.com/sokrypton/ColabFold)
4. Foldseek [paper](https://www.nature.com/articles/s41587-023-01773-0), [github](https://github.com/steineggerlab/foldseek)
5. Foldseek-multimer [paper](https://www.nature.com/articles/s41592-025-02593-7), [github](https://github.com/steineggerlab/foldseek)
6. Foldmason [paper](https://www.biorxiv.org/content/10.1101/2024.08.01.606130v1), [github](https://github.com/steineggerlab/foldmason)
7. Folddisco [paper](https://www.biorxiv.org/content/10.1101/2025.07.06.663357v1), [github](https://github.com/steineggerlab/folddisco)

## Workflow and package
1. nextflow [paper](https://www.nature.com/articles/nbt.3820), [github](https://github.com/nextflow-io/nextflow), [homepage](https://www.nextflow.io/)

## Assembler
1. spades [paper](https://www.liebertpub.com/doi/full/10.1089/cmb.2012.0021), [github](https://github.com/ablab/spades), [homepage](https://cab.spbu.ru/software/spades/)
2. trinity [paper](https://www.nature.com/articles/nbt.1883), [github](https://github.com/trinityrnaseq/trinityrnaseq)
3. flye [paper](https://www.nature.com/articles/s41587-019-0072-8), [github](https://github.com/fenderglass/Flye)
4. hifiasm [paper](https://www.nature.com/articles/s41592-020-01056-5), [github](https://github.com/chhylp123/hifiasm)
5. Raven [paper](https://www.nature.com/articles/s43588-021-00073-4), [github](https://github.com/lbcb-sci/raven)
6. Verkko [paper](https://www.nature.com/articles/s41587-023-01662-6), [github](https://github.com/marbl/verkko)

## Read aligner
1. hisat1+2 [paper](https://www.nature.com/articles/s41587-019-0201-4), [github](https://github.com/DaehwanKimLab/hisat2), [homepage](http://daehwankimlab.github.io/hisat2/)
2. star [paper](https://academic.oup.com/bioinformatics/article/29/1/15/272537), [github](https://github.com/alexdobin/STAR)
3. minimap2 [paper](https://academic.oup.com/bioinformatics/article/34/18/3094/4994778), [github](https://github.com/lh3/minimap2)
4. vsearch [paper](https://peerj.com/articles/2584/), [github](https://github.com/torognes/vsearch)
5. GraphAligner [paper](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02157-2), [github](https://github.com/maickrau/GraphAligner)
6. GraphChainer [paper](https://academic.oup.com/bioinformatics/article/39/8/btad460/7231478?login=true), [github](https://github.com/algbio/GraphChainer)
7. winnowmap2 [paper1](https://academic.oup.com/bioinformatics/article/36/Supplement_1/i111/5870473?login=true), [paper2](https://www.nature.com/articles/s41592-022-01457-8), [github](https://github.com/marbl/Winnowmap)

## Language models
1. ProtTrans [paper](https://ieeexplore.ieee.org/document/9477085), [github](https://github.com/agemagician/ProtTrans)
2. ESM3 [paper](https://www.science.org/doi/full/10.1126/science.ads0018), [github](https://github.com/evolutionaryscale/esm)
3. GenSLMs [paper](https://www.biorxiv.org/content/10.1101/2022.10.10.511571v2), [github](https://github.com/ramanathanlab/genslm)
4. Nucleotide Transformer [paper](https://www.nature.com/articles/s41592-024-02523-z), [github](https://github.com/instadeepai/nucleotide-transformer)

## Variant caller
1. freebayes [paper](https://arxiv.org/abs/1207.3907), [github](https://github.com/freebayes/freebayes)
2. pilon [paper](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0112963), [github](https://github.com/broadinstitute/pilon)
3. DeepVariant [paper](https://www.nature.com/articles/nbt.4235.epdf?author_access_token=q4ZmzqvvcGBqTuKyKgYrQ9RgN0jAjWel9jnR3ZoTv0NuM3saQzpZk8yexjfPUhdFj4zyaA4Yvq0LWBoCYQ4B9vqPuv8e2HHy4vShDgEs8YxI_hLs9ov6Y1f_4fyS7kGZ), [github](https://github.com/google/deepvariant)
4. Strelka2 [paper](https://www.nature.com/articles/s41592-018-0051-x), [github](https://github.com/Illumina/strelka)
5. Clair3 [paper](https://www.nature.com/articles/s43588-022-00387-x.epdf?sharing_token=kNdf0czJGhElEb0EB6oagdRgN0jAjWel9jnR3ZoTv0PiJdK5GreqltZHoXbqG5RzgmGodWvUVeLCJN4bGjchqsDftg2GnKA_d8pBr_SKRpJ13BBz1pMuALldqSi4zNg48pyWtmSs4ORgZmNJ1VeOjrVcoTG0y_eyXoRHpc8498U%3D), [github](https://github.com/HKU-BAL/Clair3)
6. Longshot [paper](https://www.nature.com/articles/s41467-019-12493-y), [github](https://github.com/pjedge/longshot)
7. Sniffles2 [paper1](https://www.nature.com/articles/s41592-018-0001-7), [paper2](https://www.nature.com/articles/s41587-023-02024-y), [github](https://github.com/fritzsedlazeck/Sniffles)
8. Lancet [paper](https://www.biorxiv.org/content/10.1101/2020.07.04.158063v1), [github]( https://github.com/nygenome/lancet)

## Other topics
1. Conterminator [paper](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02023-1), [github](https://github.com/martin-steinegger/conterminator)
2. VG [paper](https://www.nature.com/articles/nbt.4227), [github](https://github.com/vgteam/vg)
