---
File: README.md
Lecture: Advanced Bioinformatics 1
Professor: Martin Steinegger (martin.steinegger@snu.ac.kr)
TA: Sooyoung Cha, Victor Mihaila (steineggerlab.ta.2025@gmail.com)
Last Modified: 2025-07-25
---

# Advanced Bioinformatics 1
# The paper pool for Advanced Bioinformatics 2025

## Annotation
1. Prokka [paper](https://academic.oup.com/bioinformatics/article/30/14/2068/2390517), [github](https://github.com/tseemann/prokka)
2. Augustus [paper](https://academic.oup.com/bioinformatics/article/27/6/757/234821), [github](https://github.com/Gaius-Augustus/Augustus)
3. Maker [paper](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-491), [homepage](https://www.yandell-lab.org/software/maker.html)
4. Prodigal [paper](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-11-119), [github](https://github.com/hyattpd/Prodigal)
5. Bakta
6. DFAST
7. EggNOG
8. Alphagenome

## Genome comparison
1. mummer1-4 [paper](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005944), [github](https://github.com/mummer4/mummer)
2. mash [paper](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0997-x), [github](https://github.com/marbl/mash)
3. skani
4. Gegenees

## Multiple sequence alignment (MSA)
1. clustalomega [paper](https://www.embopress.org/doi/full/10.1038/msb.2011.75), [github](https://github.com/GSLBiotech/clustal-omega), [homepage](http://www.clustal.org/omega/)
2. mafft [paper](https://academic.oup.com/mbe/article/30/4/772/1073398), [github](https://github.com/GSLBiotech/mafft), [homepage](https://mafft.cbrc.jp/alignment/software/)
3. muscle [paper1](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-5-113), [paper2](https://academic.oup.com/nar/article/32/5/1792/2380623), [github](https://github.com/rcedgar/muscle), [homepage](https://www.drive5.com/muscle/)
4. T-Coffee
5. Kalign 3

## Homology search
1. MMseqs2 [paper](https://www.nature.com/articles/nbt.3988), [github](https://github.com/soedinglab/MMseqs2)
2. Diamond1+2 [paper1](https://www.nature.com/articles/s41592-021-01101-x), [paper2](https://www.nature.com/articles/nmeth.3176), [github](https://github.com/bbuchfink/diamond)
3. HHblits [paper](https://www.nature.com/articles/nmeth.1818), [github](https://github.com/soedinglab/hh-suite)
4. BLAST

## Transcript
1. stringtie1+2 [paper1](https://www.nature.com/articles/nbt.3122), [paper2](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1910-1), [github](https://github.com/gpertea/stringtie), [homepage](http://ccb.jhu.edu/software/stringtie/)
2. kallisto [paper](https://www.nature.com/articles/nbt.3519), [github](https://github.com/pachterlab/kallisto), [homepage](https://pachterlab.github.io/kallisto/)
3. salmon [paper](https://www.nature.com/articles/nmeth.4197), [github](https://github.com/COMBINE-lab/salmon), [homepage](https://combine-lab.github.io/salmon/)
4. RSEM

## Metagenomics
1. metaphlan3 [metaphlan1_paper](https://www.nature.com/articles/nmeth.2066), [metaphlan2_paper](https://www.nature.com/articles/nmeth.3589), [metaphlan3_paper](https://elifesciences.org/articles/65088), [github](https://github.com/biobakery/MetaPhlAn), [homepage](https://huttenhower.sph.harvard.edu/metaphlan3/)
2. kraken2 [paper](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1891-0), [github](https://github.com/DerrickWood/kraken2)
3. Kaiju
4. MetaMaps
5. Metabuli

## Phylogenetics
1. raxml [paper](https://academic.oup.com/bioinformatics/article/30/9/1312/238053), [github](https://github.com/stamatak/standard-RAxML), [homepage](https://cme.h-its.org/exelixis//web/software/raxml/index.html)
2. fasttree1,2 [paper1](https://academic.oup.com/mbe/article/26/7/1641/1128976),[paper2](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0009490), [homepage](http://www.microbesonline.org/fasttree/)
3. iqtree1,2,3 [iqtree1_paper](https://academic.oup.com/mbe/article/37/5/1530/5721363), [iqtree2_paper](https://academic.oup.com/mbe/article/37/5/1530/5721363), [iqtree3_paper](https://ecoevorxiv.org/repository/view/8916/), [github](https://github.com/iqtree/iqtree3), [homepage](http://www.iqtree.org/)
4. Unicore
5. orthofinder

## RNA structure prediction
1. viennarna [paper](https://almob.biomedcentral.com/articles/10.1186/1748-7188-6-26), [github](https://github.com/ViennaRNA/ViennaRNA), [homepage](https://www.tbi.univie.ac.at/RNA/)
2. LinearFold
3. UFold
4. MXfold2
5. RoseTTAFoldNA
6. RhoFold
7. GeoRNA

## Protein structure
1. AlphaFold2 [paper](https://www.nature.com/articles/s41586-021-03819-2), [github](https://github.com/deepmind/alphafold)
2. RoseTTAFold [paper](https://www.science.org/doi/abs/10.1126/science.abj8754), [github](https://github.com/RosettaCommons/RoseTTAFold)
4. Colabfold
5. ESMFold
6. Foldseek
7. Foldseek-multimer
8. Foldmason
9. Folddisco

## Workflow and package
1. nextflow [paper](https://www.nature.com/articles/nbt.3820), [github](https://github.com/nextflow-io/nextflow), [homepage](https://www.nextflow.io/)

## Assembler
1. spades [paper](https://www.liebertpub.com/doi/full/10.1089/cmb.2012.0021), [github](https://github.com/ablab/spades), [homepage](https://cab.spbu.ru/software/spades/)
2. trinity [paper](https://www.nature.com/articles/nbt.1883), [github](https://github.com/trinityrnaseq/trinityrnaseq)
3. flye [paper](https://www.nature.com/articles/s41587-019-0072-8), [github](https://github.com/fenderglass/Flye)
4. hifiasm [paper](https://www.nature.com/articles/s41592-020-01056-5), [github](https://github.com/chhylp123/hifiasm)
5. Velvet
6. Canu
7. Raven
8. Verkko

## Read aligner
1. hisat1+2 [paper](https://www.nature.com/articles/s41587-019-0201-4), [github](https://github.com/DaehwanKimLab/hisat2), [homepage](http://daehwankimlab.github.io/hisat2/)
2. star [paper](https://academic.oup.com/bioinformatics/article/29/1/15/272537), [github](https://github.com/alexdobin/STAR)
3. minimap2 [paper](https://academic.oup.com/bioinformatics/article/34/18/3094/4994778), [github](https://github.com/lh3/minimap2)
4. vsearch [paper](https://peerj.com/articles/2584/), [github](https://github.com/torognes/vsearch)
5. GraphAligner
6. Astra
7. GraphChainer
8. winnowmap2
9. ngmlr2

## Language models
1. ProtTrans [paper](https://ieeexplore.ieee.org/document/9477085), [github](https://github.com/agemagician/ProtTrans)
2. ESM3
3. GenSLMs
4. Nucleotide Transformer

## Variant caller
1. freebayes [paper](https://arxiv.org/abs/1207.3907), [github](https://github.com/freebayes/freebayes)
2. pilon [paper](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0112963), [github](https://github.com/broadinstitute/pilon)
3. DeepVariant
4. Strelka2
5. Clair3
6. Longshot
7. Sniffles2
8. Lancet

## Other topics
1. Conterminator [paper](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02023-1), [github](https://github.com/martin-steinegger/conterminator)
2. VG [paper](https://www.nature.com/articles/nbt.4227), [github](https://github.com/vgteam/vg)
